sections/adventure_and_skirmish_mode.tex

```diff
@@ -181,55 +184,55 @@
 
     \begin{center}
       \includegraphics[width=\linewidth]{\cards/adventure_combat.png}
-      \textbf{\textit{\textcolor{darkcandyapplered}{Adventure-Combat Card}}}
+      \imagecaption{Adventure-Combat card}
       \begin{enumerate}
-        \item[\textbf{4.}] Combat Power Level
+        \item[\textbf{4.}] Combat Power Levels
         \item[\textbf{5.}] Battle Reward
         \item[\textbf{6.}] Basic Reward
       \end{enumerate}
     \end{center}
   \end{multicols}
-  To resolve an \textbf{Adventure-Event Card} just resolve the printed effect.
+  To resolve an \textbf{Adventure-Event card} just resolve the printed effect.
   \smallskip
 
-  To resolve a \textbf{Adventure-Combat Card} follow these steps:
+  To resolve a \textbf{Adventure-Combat card} follow these steps:
   \begin{enumerate}
-    \item \textbf{Declare one Combat Power Level:}
-    You want to reach at least that power value to gain the fully reward.
+    \item \textbf{Declare one of the three available Combat Power Levels:}\index{Combat Power}
+    You want to reach at least that power value to gain the full reward.
     \item \textbf{Determine your Combat Power:}
-    Select 2 of your Units and roll 1 Attack die for each.
-    If you haven't two Units, select as many as you have.
-    Apply all Unit Attack Values and all Attack Die results to determine your final Combat Power.
+    Select 2 of your units and roll 1 Attack Die for each.
+    If you have less than two units, select as many as you have.
```

Comment:
```suggestion
    If you have fewer than two units, select as many as you have.
```
It's a little ridiculous anyway, because the only option for this is 1.

Reply:
0 units is also fewer than 2